### PR TITLE
AC Stub Test - LRAC-8462 | master

### PR DESCRIPTION
**Pull request by AfonsoHSantos**
https://issues.liferay.com/browse/LRAC-8462

### CODE REVIEW

**Comment by alexcandidos** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsList.testcase [hunk 1]
```diff
@@ -29,6 +29,15 @@ definition {
 		ACUtils.tearDownAC();
 	}
 
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8462 | Test Summary: Inform the User when there are no disabled Segments"
+	@ignore = "true"
+	@priority = "5"
+	test AssertUserIsInformedWhenThereAreNoDisabledSegments {
+
+		// TODO LRAC-8462 AssertUserIsInformedWhenThereAreNoDisabledSegments
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsList.testcase:37]: hi @AfonsoHSantos , this test is a priority should be 3 according to our test map.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsList.testcase [hunk 2]
```diff
@@ -145,4 +154,22 @@ definition {
...
+		// TODO LRAC-8460 OrderSegmentList
+
+	}
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8461 | Test Summary: Paginate Segment list"
+	@ignore = "true"
+	@priority = "4"
+	test PaginateSegmentList {
+
+		// TODO LRAC-8461 PaginateSegmentList
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsList.testcase:171]: hi @AfonsoHSantos , this test is a priority should be 3 according to our test map.

**Review by alexcandidos — CHANGES_REQUESTED** [review]
hi @AfonsoHSantos , could you please update the test priorities?

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
Hi @alexcandidos, i made the changes!

**Comment by alexcandidos** [comment]
Hi @AfonsoHSantos , Thanks for working on it!

The code looks good to me!

**Comment by alexcandidos** [comment]
ci:forward

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**